tests/SprykerEcoTest/Zed/Adyen/Business/BaseSetUpTest.php

```diff
@@ -414,4 +445,24 @@ protected function createRedirectResponseTransfer(OrderTransfer $orderTransfer):
 
         return $redirectResponseTransfer;
     }
+
+    /**
+     * @param int $idSalesOrder
+     *
+     * @return string
+     */
+    protected function getPaymentAdyenReference(int $idSalesOrder): string
+    {
+        return sprintf(static::PAYMENT_ADYEN_REFERENCE, $idSalesOrder);
```

Comment:
please remove protected constant and put string with pattern directly to sprintf arguments list